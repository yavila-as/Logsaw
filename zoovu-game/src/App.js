import React, { useState } from 'react'
import './App.css';
import WelcomeScreen from '../src/containers/WelcomeScreen';
import GameScreen from '../src/containers/GameScreen';

const App = () => {
  const [submited, setSubmited ] = useState(false);
  const [name, setName ] = useState('');
  const handleSubmit = () => {
    setSubmited(true);
  };

  return (
    <div className="App">
      {submited ? (
        <GameScreen  userName={name}  userName={name} />
      ) : (
          <header className="App-header">
            <WelcomeScreen onSubmit={handleSubmit} userName={name} setUserName={setName}/>
          </header>
      )}
    </div>
  )
}

export default App;

