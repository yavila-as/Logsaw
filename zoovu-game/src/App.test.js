import { render, screen } from '@testing-library/react';
import App from './App';
import React from 'react';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
