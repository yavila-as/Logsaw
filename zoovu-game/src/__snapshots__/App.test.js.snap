// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text"
      data-testid="boxWelcome"
      disabled={false}
      display="flex"
      flexwrap="wrap"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        <form
          autoComplete="off"
          noValidate={true}
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
          >
            <p
              className="MuiTypography-root MuiTypography-body1"
              data-testid="welcomeUser"
            >
              Hello friend, tell me your name...
            </p>
            <div
              className="MuiFormControl-root MuiTextField-root makeStyles-textField-1 MuiFormControl-marginNormal"
            >
              <label
                className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
                data-shrink={false}
                htmlFor="standard-name"
                id="standard-name-label"
              >
                Your name here
              </label>
              <div
                className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
                onClick={[Function]}
              >
                <input
                  aria-invalid={false}
                  autoFocus={false}
                  className="MuiInputBase-input MuiInput-input"
                  disabled={false}
                  id="standard-name"
                  onAnimationStart={[Function]}
                  onBlur={[Function]}
                  onChange={[Function]}
                  onFocus={[Function]}
                  required={false}
                  type="text"
                  value=""
                />
              </div>
            </div>
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-2"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <p
                  className="MuiTypography-root MuiTypography-body1"
                >
                  Let's go
                </p>
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.01 11H4v2h12.01v3L20 12l-3.99-4z"
                  />
                </svg>
              </span>
            </button>
          </div>
        </form>
      </span>
    </button>
  </header>
</div>
`;
